doc/nrf/applications.rst

```diff
@@ -5,6 +5,7 @@ Applications
 
```

Comment:
```suggestion
The |NCS| provides a set of example applications that show how to implement typical user scenarios using Nordic Semiconductor devices.
```
Maybe rewrite the next sentences a bit to avoid duplication. But the main point here is that the applications implement full use cases. Mentioning the fully integrated software stack is important of course, but it's not the most relevant point.

Reply:
Thanks. Tried to reduce duplication.